server/templates/obslist.html

```diff
@@ -68,6 +68,8 @@ <h3>Observations</h3>
         <th class="text-center"><abbr title="Two Line Element">TLE</abbr></th>
         <th class="text-center">Rating</th>
         <th>Station</th>
+        <th>Frequency</th>
+        <th>Protocol</th>
```

Comment:
I think that the protocol shouldn't be visible on the main list.

- The protocol Is little related to the frequency because the satellites use one protocol per frequency.
- The protocol may be unknown, but the frequency always is known.
- User may be unsure what protocol is used. For example, when the embedded metadata is received.

Reply:
I can easily hide it, but I was thinking that it's useful to compare APT to LRPT for example. People can draw conclusions from that. I was thinking that long term we would expose all of them, so you could filter and compare various scenarios, APT vs LRPT, reception with and without LNA etc. Also, having the protocol clickable (link to wikipedia) would have some educational value. Finally, the third argument is about emphasizing new protocols, which I very much hope we'll see soon.

So, does that sound convincing? If not, I'll remove it.

Reply:
Ok, it makes sense. But we should be careful that the table isn't too wide.